app/javascript/gobierto_data/webapp/components/sets/VisualizationsItem.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <div class="gobierto-data-sql-editor">
+    <template v-if="(privateVisualizations.length && items) || (publicVisualizations.length && items)">
```

Comment:
when you think the logic becomes complex, a computed property will solve that out.